app/src/main/java/com/example/una/CharityHomeActivity.java

```diff
@@ -0,0 +1,24 @@
+package com.example.una;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import com.example.una.models.Charity;
+
+import org.parceler.Parcels;
+
+public class CharityHomeActivity extends AppCompatActivity {
+
+    Charity charity;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_charity_home);
+
+        charity = Parcels.unwrap(getIntent().getParcelableExtra("charity"));
+    }
+
+    // TODO: recyclerview of all the charities posts (challenges and notifications)
```

Comment:
Do you think we should eventually have separate screens for challenges and notifications for charity users?

Reply:
If so, could somehow reuse the challenges recycler view that we currently have in the Impact Fragment (and maybe query differently to display 1. all charities' challenges and 2. challenges that donors have set up for the logged in charity - using tabs maybe???)

Reply:
Not sure if we want them separated though